parm/wflow/prep.yaml

```diff
@@ -29,8 +29,12 @@ task_make_grid:
 
 task_make_orog:
   <<: *default_task
-  command: '&LOAD_MODULES_RUN_TASK; "make_orog" "&JOBSdir;/JREGIONAL_MAKE_OROG"'
+  command: !cycstr 'source &USHdir;/load_modules_wflow.sh hera ; &SCRIPTSdir;/make_orog.py
```

Comment:
Should I update this line as well in my open PR to `{{ user.MACHINE }}` instead of `hera`?

Reply:
Yes please, thank you!